plugins/fastem_roa_import_export.py

```diff
@@ -0,0 +1,341 @@
+the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+Public License for more details.
+
+You should have received a copy of the GNU General Public License along with Odemis. If not,
+see http://www.gnu.org/licenses/.
+"""
+
+import csv
+from enum import Enum
+import logging
+import wx
+from wx.lib.embeddedimage import PyEmbeddedImage
+
+from odemis.gui.plugin import Plugin
+
+
+IMPORT_IMAGE = PyEmbeddedImage(
+    b"iVBORw0KGgoAAAANSUhEUgAAABgAAAAVCAYAAABc6S4mAAAAqUlEQVQ4T2NkwA3+45FDlmoA"
+    b"chqBGKSeEV0PhgCSAnIsAGlXB+JbMHNoYQGK2bS0AGQRI60twIwUCuIAa5qghg/AQYErxVHL"
+    b"ApwpmhwLbgBN0yAyj5AVBzBHEZVPyPEBsh6CllBqASiklgBxNL5IxuUKfJZjMw+rOSBDaG4B"
+    b"Lt+RYjHOuKBGHOCNaEosaAd6vYJQfiDXAoLJk5r1AV5PkOMDQqGCIk9zCwB4bSsHCFUhIAAA"
+    b"AABJRU5ErkJggg=="
+)
+
+
+EXPORT_IMAGE = PyEmbeddedImage(
+    b"iVBORw0KGgoAAAANSUhEUgAAABgAAAAVCAYAAABc6S4mAAAAjUlEQVQ4T9WUgQqAIAxE51f3"
+    b"DX11qbRYhd7NHFQgBJ4+79xMIrLlgb50CIpW/9GaOl/EXoCuCwXQkFEHenoY11sAdOIFULlb"
+    b"EQuAJ22RPYDeHk1nswFLJq2jETEOtKfO6prtwDZthUQC6v4MADVT96n5BAA1l3V4d0NFNAoI"
+    b"veTQMr0UBXPJnoge2v8DdsqAJBTcjQywAAAAAElFTkSuQmCC"
+)
+
+
+class CSVFieldnames(Enum):
+    PROJECT_NAME = "project_name"
+    ROA_NAME = "roa_name"
+    ROA_COORDINATES_L = "roa_coordinates_l"
+    ROA_COORDINATES_T = "roa_coordinates_t"
+    ROA_COORDINATES_R = "roa_coordinates_r"
+    ROA_COORDINATES_B = "roa_coordinates_b"
+    ROC_2_NAME = "roc_2_name"
+    ROC_2_COORDINATES_L = "roc_2_coordinates_l"
+    ROC_2_COORDINATES_T = "roc_2_coordinates_t"
+    ROC_2_COORDINATES_R = "roc_2_coordinates_r"
+    ROC_2_COORDINATES_B = "roc_2_coordinates_b"
+    ROC_3_NAME = "roc_3_name"
+    ROC_3_COORDINATES_L = "roc_3_coordinates_l"
+    ROC_3_COORDINATES_T = "roc_3_coordinates_t"
+    ROC_3_COORDINATES_R = "roc_3_coordinates_r"
+    ROC_3_COORDINATES_B = "roc_3_coordinates_b"
+
+    @staticmethod
+    def list():
+        return list(map(lambda c: c.value, CSVFieldnames))
+
+
+class ImportExportROAPlugin(Plugin):
+    name = "Import Export ROA"
+    __version__ = "1.0"
+    __author__ = "Nandish Patel"
+    __license__ = "GPLv2"
+
+    def __init__(self, microscope, main_app):
+        super().__init__(microscope, main_app)
+
+        # It only makes sense if the FASTEM acquisition tab is present
+        try:
+            self._acquisition_tab = main_app.main_data.getTabByName("fastem_acqui")
+        except LookupError:
+            logging.debug(
+                "Not loading Import Export ROAs tool since acquisition tab is not present"
+            )
+            return
+
+        self.addMenu("Help/Development/Import Export ROAs", self.import_export_roa)
+
+    def import_export_roa(self):
+        """Help/Development/Import Export ROAs callback."""
+        ImportExportTool().show(
+            project_list_controller=self._acquisition_tab._project_list_controller
+        )
+
+
+class ImportExportTool(object):
```

Comment:
That looks a very complex class, to just be initialized, called .show() once, and immediately be discarded.
I think you can move the `init()` to the plugin `__init__()`. Then, you can move the code of `show()` to `import_export_roa()`.